packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -811,8 +812,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this.reSubmit;
     }
 
+    public get disposeFn(): (error?: ICriticalContainerError) => void {
+        return this.context.disposeFn ?? this.context.closeFn;
```

Comment:
Maybe add a comment something like "In old loaders with no separate dispose functionality, close is equivalent (but will emit readonly events)"